### PR TITLE
VIH-8967 - Setup Notification API to SDS

### PR DESCRIPTION
**Pull request by v1ferrarij**
### JIRA link (if applicable) ###

VIH-8967

### Change description ###

Setup Notification API to SDS

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

**Review by thomaswatsonv1 — DISMISSED** [review]
Looks good

File: azure-pipelines.sds.pr-release.yml [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+              chartName: ${{ variables.chartName }}
+              chartPath: ${{ variables.chartPath }}
+              subscriptionName: "${{ variables.subscriptionName }}"
+              aksResourceGroupName: "$(aksCluster.resourceGroup)"
+              aksClusterName: "$(aksCluster.name)"
+              dnsSubscriptionName: "${{ variables.dnsZoneSubscription }}"
+              dnsResourceGroupName: "${{ variables.dnsZoneResourceGroup }}"
+
+      - ${{ if eq(variables.run_acceptance_tests, true) }}:
+        - job: AcceptanceTestsDev
```
Comment (chris-pateman) [inline azure-pipelines.sds.pr-release.yml:249]: Have you added all the acceptance test settings from here
https://github.com/hmcts/vh-notification-api/blob/71aa3a9a0b179fca6da15fe222b80ebe3a6e4639/azure-pipelines.yml#L42

File: azure-pipelines.sds.release.yml [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+
+#####################################################
+# Run Entity Framework Staging. #####################
+
+  - ${{ each env in parameters.environment }}:
+    - stage: Run_Entity_Framework_${{ env }}
+      ${{ if eq(env, 'Prod') }}:
+        condition: and(succeeded(), contains(variables['Build.SourceBranch'], 'refs/heads/release'))
+      ${{ else }}:
+        condition: and(succeeded(), eq(variables['Build.SourceBranchName'], 'VIH-8967')) # change me
```
Comment (chris-pateman) [inline azure-pipelines.sds.release.yml:91]: change me

File: charts/vh-notification-api/values.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      resourceGroup: vh-infra-core-{{ .Values.global.environment }}
+      usePodIdentity: true
+      secrets:
+        - azuread--clientsecret
+        - azuread--clientid
+        - notifyconfiguration--apikey
+        - notifyconfiguration--callbacksecret
+  environment:
+    ASPNETCORE_URLS: http://+:8080
+    AZUREAD__TENANTID: "fb6e0e22-0da3-4c35-972a-9d61eb256508"
```
Comment (chris-pateman) [inline charts/vh-notification-api/values.yaml:32]: don't need this as it comes from the KV

**Review by chris-pateman — CHANGES_REQUESTED** [review]

**Review by thomaswatsonv1 — APPROVED** [review]
Looks good!

**Pull request merged**